mycroft/skills/__main__.py

```diff
@@ -213,25 +209,20 @@ def main():
     reset_sigint_handler()
     # Create PID file, prevent multiple instances of this service
     mycroft.lock.Lock('skills')
+    # Connect this Skill management process to the Mycroft message bus
     config = Configuration.get()
     # Set the active lang to match the configured one
     set_active_lang(config.get('lang', 'en-us'))
-
-    # Connect this process to the Mycroft message bus
```

Comment:
Could have been a merge issue when I pulled the your changes in.  Let me know where to look for how this should be and I will change it.

Reply:
I think the problem may be that this is a continuation of the messagebus cleanup you did since the branch contains all those commits, maybe you can create a fresh branch from dev and cherrypick the skill manager update...

Something like
```
git checkout refactor-skill-manager
git checkout -b refactor-skill-manager-bak
git branch -D refactor-skill-manager
git checkout -b refactor-skill-manager dev
git cherry-pick 69d442f18205f47e535621b9c2015f7e24100313..refactor-skill-manager-bak
```

May be some issues since there are merge commits in the range.

Reply:
Or basically revert all changes done in `__main__.py`

Reply:
I just merged origin/dev into this branch and pushed the result.  Let me know if this fixed the issue.

Reply:
Nope the removes are still there... (see the diff of __main__.py)

Reply:
I believe this is addressed now.